src/lsst/cmservice/db/queue.py

```diff
@@ -173,7 +174,7 @@ async def get_create_kwargs(
             node = await Job.get_row_by_fullname(session, fullname)
             ret_dict["j_id"] = node.id
         elif node_level == LevelEnum.script:
-            node = await Script.get_row_by_fullname(session, fullname)
+            node = await Script.get_row_by_fullname(session, fullname[7:])
```

Comment:
Okay, now I'm going to recommend a named constant. :-)   While you are at it, get the "300" a few lines above, what the heck...